src/CalcManager/winerror_cross_platform.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
```

Comment:
Should not be needed anymore, once the typedef below is gone.

Reply:
@danbelcher-MSFT I forgot to open it again after reverting the change that originally fixed it, my mistake.

Currently, however, the include is required since the `SUCCEEDED` depends on `HRESULT`, which in turn depends on `uint32_t`.